velox/dwio/parquet/reader/ParquetReader.h

```diff
@@ -126,6 +126,10 @@ class ParquetReader : public dwio::common::Reader {
 
   size_t numberOfRowGroups() const;
 
+  thrift::CompressionCodec::type getCompressionKind(
+      int32_t rowGroupIndex,
```

Comment:
Do you want to use defaults = 0 here so that it matches the DWRF reader API when not providing groupIndex or colIndex?

Reply:
Parquet and Dwrf differ in terms of where the compression information is stored. For Dwrf, it is stored in the [Postscript](https://orc.apache.org/specification/ORCv1/) which is common for all the stripes.
For Parquet, the compression information is specific to the column in a RowGroup. In practice, all RowGroup columns use the same compression.
IMO we should not add a default here to ensure the client knows the compression information is tied to a column in a RowGroup.

Reply:
Let's go with Christian's suggestion for the ParquetReader API.
The ReaderBase API can have the arguments. See the example in the other comment.